### ISSUE TITLE
libman.json : error LIB000: An unknown exception occurred after upgrading from Microsoft.Web.LibraryManager.Build from 2.0.96 to 2.1.50

### ISSUE THREAD

**Issue by cdemi** [aspnet/LibraryManager#574]
Build fails on Docker image `mcr.microsoft.com/dotnet/core/sdk:3.1-buster` after package upgrade from 2.0.96 to 2.1.50 because one or more libman libraries failed to restore

Current `libman.json`:
```
{
  "version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "provider": "cdnjs",
      "library": "twitter-bootstrap@4.4.0",
      "destination": "wwwroot/lib/twitter-bootstrap/",
      "files": [
        "css/bootstrap.css",
        "js/bootstrap.js"
      ]
    },
    {
      "provider": "cdnjs",
      "library": "popper.js@2.3.3",
      "destination": "wwwroot/lib/popper.js/",
      "files": [
        "umd/popper.js"
      ]
    },
    {
      "provider": "cdnjs",
      "library": "jquery@3.5.0",
      "destination": "wwwroot/lib/jquery/",
      "files": [
        "jquery.js"
      ]
    }
  ]
}
```

Build Error:
```
2020-04-15T23:10:24.2235627Z   Restore operation started...
2020-04-15T23:10:24.6515837Z libman.json : error LIB000: An unknown exception occurred [/src/NewsletterCurator.Web/NewsletterCurator.Web.csproj]
2020-04-15T23:10:24.6522938Z   
2020-04-15T23:10:24.6524619Z   One or more libraries failed to restore
```

**Comment by Mobrockers** [comment]
I get the same after upgrading in an azure devops pipeline. Build-restore works fine in visual studio.

Our build pipeline is public so if it helps you can find the build logs here: https://dev.azure.com/stryker-mutator/Stryker/_build/results?buildId=4273&view=results

**Comment by jimmylewis** [comment]
I was able to repro this when using the `DotNetCoreCLI@2` task to build a solution, but not when using `VSBuild@1`.  (Thanks @Mobrockers for the sample, that saved some time for sure).  Still investigating, but just wanted to let you know I'd verified the issue.

**Comment by cdemi** [comment]
Just to give you some more context, in my case I am using [this Dockerfile](https://github.com/cdemi/NewsletterCurator/blob/dependabot/nuget/Microsoft.Web.LibraryManager.Build-2.1.50/NewsletterCurator.Web/Dockerfile) and the error is generated during `dotnet publish "NewsletterCurator.Web.csproj" /p:Version=$VERSION -c Release -o /app/publish`.

This is my [libman.json](https://github.com/cdemi/NewsletterCurator/blob/master/NewsletterCurator.Web/libman.json)

If you need additional information, tell me

**Comment by EdiWang** [comment]
Same issue here with Azure DevOps

![image](https://user-images.githubusercontent.com/3304703/79678298-643b4800-822c-11ea-9a55-1f52202f7f83.png)

**Comment by Ascathor** [comment]
Heya, 
same situation here using dotnetCLI.

Reverting to 2.0.96 fixed this.

**Comment by jimmylewis** [comment]
This should be fixed by #575, just waiting now on our release process to publish the updated package.

**Comment by jimmylewis** [comment]
2.1.76 is live now.
